definitions/npm/@tanstack/react-query_v5.x.x/flow_v0.83.x-/react-query_v5.x.x.js

```diff
@@ -0,0 +1,118 @@
+declare module "@tanstack/react-query" {
+  declare type QueryFunctionContext = { [key: string]: any };
+
+  declare type UseQueryOptions<T> = {|
```

Comment:
Nit just to be consistent with the rest of the types
```suggestion
  declare type UseQueryOptions<TData> = {|
```